_posts/linguistic-limitation.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+header_image: /img/posts/linguistic-limitation/header.jpg
+title: "A linguistic limitation"
+author: Mauro Servienti
+synopsis: "The language we use is crucial to get the right message through. The words we use have to be as specific as we can to avoid misunderstandings. It seems we have space for improvement, and we should be doing a better job."
+tags:
+- ubiquitous language
+---
+
+I have a feeling this is going to be a short one. I have no answers, only doubts.
+
+A couple of weeks ago, I was chatting with a friend, discussing some architectural designs. At a certain point, we mentioned publishing an event as a solution to one of the issues. All of a sudden, we stopped and asked the question: what kind of event? A domain event or a pub/sub-one?
+
+As a result, we stopped discussing the initial architectural concern and moved to a completely different topic: the ubiquitous language or the lack of a proper one.
+
+Martin Fowler [defines the ubiquitous language](https://www.martinfowler.com/bliki/UbiquitousLanguage.html) as follows:
+
+> Ubiquitous Language is the term Eric Evans uses in Domain Driven Design for the practice of building up a common, *rigorous language between developers and users*. This language should be based on the Domain Model used in the software - hence the need for it to be rigorous, since software doesn't cope well with ambiguity.
+
+(emphasis is mine)
```

Comment:
```suggestion
Martin Fowler [defines the ubiquitous language](https://www.martinfowler.com/bliki/UbiquitousLanguage.html) as follows (emphasis is mine):

> Ubiquitous Language is the term Eric Evans uses in Domain Driven Design for the practice of building up a common, *rigorous language between developers and users*. This language should be based on the Domain Model used in the software - hence the need for it to be rigorous, since software doesn't cope well with ambiguity.
```